components/custom_status/__snapshots__/expiry_menu.test.tsx.snap

```diff
@@ -19,8 +19,8 @@ exports[`components/custom_status/expiry_menu should match snapshot 1`] = `
           className="MenuWrapper statusExpiry__menu"
           onClick={[Function]}
         >
-          <span
-            className="expiry-wrapper expiry-selector"
+          <button
+            className="style--none expiry-wrapper expiry-selector"
           >
```

Comment:
can we make this single line please?

```suggestion
          <button className="style--none expiry-wrapper expiry-selector">
```

Reply:
This is a generated snapshot, I can't update the layout because it will get overwritten